deployability/modules/testing/tests/test_agent/test_basic_info.py

```diff
@@ -67,19 +67,32 @@ def setup_test_environment(wazuh_params):
-        if HostInformation.get_os_type(agent_params) == 'linux':
-            assert HostInformation.dir_exists(agent_params, WAZUH_ROOT), logger.error(f'The {WAZUH_ROOT} is not present in {HostInformation.get_os_name_and_version_from_inventory(agent_params)}')
-        elif HostInformation.get_os_type(agent_params) == 'macos':
-            assert HostInformation.dir_exists(agent_params, '/Library/Ossec'), logger.error(f'The /Library/Ossec is not present in {HostInformation.get_os_name_and_version_from_inventory(agent_params)}')
+        path_to_check = ''
+        os_type = HostInformation.get_os_type(agent_params)
+
+        if os_type == 'linux':
+            path_to_check = WAZUH_ROOT
+        elif os_type == 'windows':
+            path_to_check = WINDOWS_ROOT_DIR
+        elif os_type == 'macos':
+            path_to_check = MACOS_ROOT_DIR
+
+        assert HostInformation.dir_exists(agent_params, path_to_check), logger.error(f'The {path_to_check} is not present in {HostInformation.get_os_name_and_version_from_inventory(agent_params)}')
 
         expected_condition_func = lambda: 'active' == WazuhAgent.get_agent_status(wazuh_api, agent_names)
         Waits.dynamic_wait(expected_condition_func, cycles=20, waiting_time=30)
-        assert HostInformation.get_os_version_from_inventory(agent_params) in WazuhAgent.get_agent_os_version_by_name(wazuh_api, agent_names), logger.error('There is a mismatch between the OS version and the OS  version of the installed agent')
-        if HostInformation.get_os_type(agent_params) == 'linux':
-            assert HostInformation.get_os_name_from_inventory(agent_params) in WazuhAgent.get_agent_os_name_by_name(wazuh_api, agent_names).replace(' ', ''),  logger.error('There is a mismatch between the OS name and the OS name of the installed agent')
-        elif HostInformation.get_os_type(agent_params) == 'macos':
-            assert 'macos' in WazuhAgent.get_agent_os_name_by_name(wazuh_api, agent_names).replace(' ', ''),  logger.error('There is a mismatch between the OS name and the OS name of the installed agent')
 
+        if not os_type == 'windows':
+            assert HostInformation.get_os_version_from_inventory(agent_params) in WazuhAgent.get_agent_os_version_by_name(wazuh_api, agent_names), logger.error('There is a mismatch between the OS version and the OS  version of the installed agent')
+
+        if os_type == 'macos':
+            os_name = 'macos'
+        elif os_type == 'windows':
+            os_name = 'windows'
+        elif os_type == 'linux':
+            os_name = HostInformation.get_os_name_from_inventory(agent_params)
+
+        assert os_name in WazuhAgent.get_agent_os_name_by_name(wazuh_api, agent_names).replace(' ', ''),  logger.error('There is a mismatch between the OS name and the OS name of the installed agent')
 
 def test_wazuh_version(wazuh_params):
     for agent_names, agent_params in wazuh_params['agents'].items():
```

Comment:
The statements inside the loop don't use the `agent_names` variable. Could you replace it with _?

Reply:
Done in  https://github.com/wazuh/wazuh-qa/pull/5310/commits/65ca9b4687cd5d98cb49b609aa5d1f9a662d666c